thavalon-webapp/src/styles/gameStyles/interactionStyles/missionManager.scss

```diff
@@ -0,0 +1,86 @@
+@import "../gameGlobals.scss"; 
+
+.mission-card-button-row {
+    width: 90%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+%mission-button {
+    @extend %game-button;
+    font-size: 0.8rem;
+    color: white;
+    padding: 2px 5px;
+}
+
+.mission-button-success {
+    @extend %mission-button;
+    background-color: $player-selected-color-primary;
+    &:hover {
+        background: rgba($player-selected-color-primary, $button-hover-opacity);
+    }
+}
+
+.mission-button-fail {
+    @extend %mission-button;
+    background-color: $button-red;
+    &:hover {
+        background: rgba($button-red, $button-hover-opacity);
+    }
+}
+
+.mission-button-reverse {
+    @extend %mission-button;
+    background-color: $button-orange;
+    &:hover {
+        background: rgba($button-orange, $button-hover-opacity);
+    }
+}
+
+.mission-button-qb {
+    @extend %mission-button;
+    background-color: $button-blue;
+    display: none;
```

Comment:
If we make this `visibility: hidden` instead, it'll render an invisible QB button that we can show on hover.

Reply:
I think that'll mess with spacing, since it'll be there but not be visible. Showing a button on hover is too easy to find imo